include/Core/Base/Mixin/Renderer.php

```diff
@@ -116,6 +116,23 @@ protected function split_post_type_and_status( $str ) {
 		return $type_status;
 	}
 
+	/**
+	 * Render content settings.
+	 */
+	protected function render_content_settings() {
```

Comment:
Rename this function to `render_post_reassign_settings`

Reply:
Fixed.